src/data/nat/multiplicity.lean

```diff
@@ -97,6 +91,40 @@ lemma multiplicity_factorial {p : ℕ} (hp : p.prime) :
   ... = (∑ i in Ico 1 b, (n + 1) / p ^ i : ℕ) :
     congr_arg coe $ finset.sum_congr rfl (by intros; simp [nat.succ_div]; congr)
 
+/-- The multiplicity of `p` in `(p(n+1))!` is one mor more than that of `n!`. -/
```

Comment:
The docstring does not really correspond to the content of the lemma (and I don't know what `mor` means).

Reply:
Thanks, fixed.